includes/Core/Admin/Dashboard.php

```diff
@@ -79,6 +79,8 @@ function () {
 
 				// Enqueue scripts.
 				$this->assets->enqueue_asset( 'googlesitekit-wp-dashboard' );
+				$this->assets->enqueue_asset( 'googlesitekit-datastore-site' );
```

Comment:
Weirdly enough: this never runs. I don't see the asset enqueued in the network requests, and then code inside never loads. Am I doing something wrong here? I'm a bit lost 😓

Reply:
This will be enqueued on the WP dashboard `wp-admin/index.php` not the Site Kit dashboard. To do that, add it as a dependency of `googlesitekit-dashboard` 👍